client/src/components/Signup.js

```diff
@@ -18,15 +19,15 @@ import { createTheme, ThemeProvider } from '@mui/material/styles';
 
 const theme = createTheme();
 
-const Signup = ({ setAlert }) => {
+const Signup = ({ setAlert, signUp }) => {
 	const [formData, setFormData] = useState({
 		userName: '',
 		email: '',
 		password: '',
 		password_confirm: '',
 	});
 
-	const { name, email, password, password_confirm } = formData;
+	const { userName, email, password, password_confirm } = formData;
```

Comment:
Do we need to change in db models too? It will send back to db, if it doesn't match will it return error ?

Reply:
if you look into the db model, you can see the property is userName